web/packages/teleterm/src/ui/TopBar/Identity/IdentitySelector/IdentitySelector.tsx

```diff
@@ -49,10 +51,23 @@ export const IdentitySelector = forwardRef<
       ref={ref}
       onClick={props.onClick}
       title={title}
+      css={`
+        position: relative;
+      `}
     >
       {isSelected ? (
         <Box>
           <UserIcon letter={props.userName[0]} />
+          {props.deviceTrustStatus === 'requires-enrollment' && (
+            <ConnectionStatusIndicator
+              status={'warning'}
+              css={`
+                position: absolute;
+                top: 1.5px;
+                right: 1.5px;
```

Comment:
I wanted to check if this is consistent with the other indicators in the top bar and… ggs we'll get them next time.

<img width="923" alt="top-bar" src="https://github.com/user-attachments/assets/53cf9c68-b896-464b-8cad-e0e7090732ba">

Reply:
I placed it to fit in the border of the Profiles button :/ 

But in general, we should replace this icon, it's too detailed for its size and transparent, making it difficult to place over other elements. Maybe I will find time for this when redesigning the profile switcher.